.changeset/warm-pugs-divide.md

```diff
@@ -0,0 +1,35 @@
+---
+'@logto/core': minor
+'@logto/integration-tests': minor
+'@logto/phrases': minor
+'@logto/schemas': minor
+'@logto/connector-kit': minor
+---
+
+add account api
+
+Introduce the new Account API, designed to give end users direct API access without needing to go through the Management API, here is the highlights:
+
+1. Direct access: The Account API empowers end users to directly access and manage their own account profile without requiring the relay of Management API.
+2. User profile and identities management: Users can fully manage their profiles and security settings, including the ability to update identity information like email, phone, and password, as well as manage social link connections. May also support MFA and SSO in the future.
+3. Global access control: Admin has full, global control over access settings, can customize each fields.
+4. Seamless authorization: Authorizing is easier than ever! Simply use client.getAccessToken() to obtain an access token and attach it to the Authorization header.
```

Comment:
this sounds confusing. better to change to a more clear version

Reply:
updated